frontend/talentsearch/resources/js/components/skills/SkillChecklist.tsx

```diff
@@ -117,29 +119,31 @@ const Category: React.FunctionComponent<CategoryProps> = ({
 };
 
 export interface SkillChecklistProps {
-  skillFamilies: SkillFamily[];
+  skills: Skill[];
```

Comment:
If the only thing this component does with skills is to invert them into skillFamilies, doesn't it make more sense to accept skillFamilies? The inversion can happen in a parent component, if necessary. And if we want to use the checklist, and already have skillFamilies for some reason, no extra processing required.

Reply:
Yeah, you're right - this is much more natural.  Great suggestion!